garbage-collection/garbage-collection-deployment.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: hello
+spec:
+  schedule: "@weekly"
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          containers:
+          - name: garbage-collection
+            image: argo-ml/latest
```

Comment:
this image doesn't exist. We may want to publish images, but then it'd be something like github/argo-ml:latest.